SparkPost/2web2ui
# SA-1734: Saved Reports feedback

### What Changed
 - Saving new report sets report in typeahead
 - Saving report updates the edit/save report information with up to date information
 - Deleting a report removes from typeahead (if the report in typeahead is the same report)
 - Popover changed top be on the top left instead of bottom left (pending UX approval)

### How To Test
 - Go to report builder
 - Verify saving new report sets in typeahead
 - Verify that clicing edit/save report populates information correctly
 - Click on View All Reports
 - Click on the action for a report and delete report that is in the typeahead
 -Verify after deleting, the report is removed from the typeahead

### To Do
